modin/pandas/dataframe.py

```diff
@@ -3825,7 +3825,18 @@ def sortlevel(
         )
 
     def squeeze(self, axis=None):
-        return self._default_to_pandas_func(pandas.DataFrame.squeeze, axis=axis)
+        # Checks for 1x1 DF, passes into squeeze with approproate ndim
+        if (
+            self._query_compiler.data.shape[0] == 1
+            and self._query_compiler.data.shape[1] == 1
+        ):
+            return self._query_compiler.squeeze(2, axis)
```

Comment:
```suggestion
            return self._query_compiler.squeeze(0, axis)
```
Since 1x1 should be dim 0